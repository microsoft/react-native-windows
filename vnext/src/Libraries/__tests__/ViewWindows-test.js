
'use strict';

const React = require('react');
const ReactTestRenderer = require('react-test-renderer');

const {ViewWindows} = require('../Components/View/ViewWindows');

describe('ViewWindows', () => {
  it('renders basic view', () => {
    const component = ReactTestRenderer.create(
      <ViewWindows/>
    );
    expect(component).toMatchSnapshot();
  });
});