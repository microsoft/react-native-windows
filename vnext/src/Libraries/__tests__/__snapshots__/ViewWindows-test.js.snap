// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ViewWindows renders basic view 1`] = `<View />`;
